participant/app/routes/$localAgency/recertify/upload.tsx

```diff
@@ -57,22 +413,82 @@ export default function Upload() {
     fileInputRef.current?.files.forEach((value) => {
       data.append("documents", value);
     });
-    formSubmit(data, { method: "post", encType: "multipart/form-data" });
+    // Chrome throws an error if the form is empty
+    if (!data.has("documents")) {
+      data.append("documents", "");
+    }
+    formSubmit(data, {
+      method: "post",
+      encType: "multipart/form-data",
+      action: location.pathname,
+    });
   };
   return (
     <div>
       <h1>{t("Upload.title")}</h1>
+      <div>{t("Upload.intro")}</div>
+      {documentProofElements}
+      <Form method="get" id="previousFiles" name="previous-uploads-form">
```

Comment:
Why is this form method `get` and not `post`?

Reply:
A `post` would cause a page reload; I could work around that with javascript i guess?

Reply:
Oh I see. I think it's better for this to be a `get` in that case. I wasn't sure what it would leak in the url query, but it's just the filename, which I think is not sensitive.

Reply:
.. And the cookie (submissionID) is needed to identify the file, so just skivving the URL and replaying it wouldn't delete someone's file.
I think you'd need to mitm the cookie while the session was alive (30 min)